src/py/mat3ra/notebooks_utils/pyodide/packages/mace.py

```
@@ -1,4 +1,44 @@
+    """
+    Fix MACE's torch_geometric import order issues in Pyodide.
+
+    In Pyodide, torch_geometric.data may not be set during circular imports.
+    Pre-importing ensures the attribute is available when MACE needs it.
+    """
+    try:
+        torch_geometric = import_module("mace.tools.torch_geometric")
+        torch_geometric_data = import_module("mace.tools.torch_geometric.data")
+        torch_geometric.data = torch_geometric_data
+        print("✓ MACE tools patches applied")
+    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)